app/commands/markdown/render_markdown.rb

```diff
@@ -0,0 +1,18 @@
+class Markdown::Render
```

Comment:
```suggestion
class Markdown::RenderMarkdown
```

Reply:
This file shouldn't even be here. Whoops